modules/preloader/include/mistAPI.js

```diff
@@ -159,20 +157,23 @@ module.exports = () => {
                 const filteredId = prefix + filterId(id);
                 queue.push({ action: 'selectMenu', id: filteredId });
 
-                for (var e in this.entries) {
+                for (let e in this.entries) {
                     if ({}.hasOwnProperty.call(this.entries, e)) {
                         this.entries[e].selected = (e === filteredId);
```

Comment:
The other ones need to set `selected = false`, you only seem to set the new one to true, which means multiple ones have true now.